### PR TITLE
fix(client-app): request basic auth placeholder

### PR DESCRIPTION
**Pull request by antlio**
this pr updates the basic auth request password placeholder.

### CODE REVIEW

File: packages/client-app/src/components/DataTable/DataTable.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@ defineProps<{
 }>()
 </script>
 <template>
-  <div
+  <form
```
Comment (antlio) [inline packages/client-app/src/components/DataTable/DataTable.vue:9]: setting that back to comply with good practice and fix error log in chrome, any reason to have removed it @cameronrohani?

Comment (marclave) [reply packages/client-app/src/components/DataTable/DataTable.vue:9]: the table doesn't show up for form data or other body elements when it's a form :)

Comment (antlio) [reply packages/client-app/src/components/DataTable/DataTable.vue]: oh make sense no form within table, updated 🙏

**Review by marclave — APPROVED** [review]
nice work @antlio ! tested and is great ✨

**Pull request merged**